pkg/kustomize/kustomize.go

```diff
@@ -264,6 +264,14 @@ func KustomizationFor(ctx *quaycontext.QuayRegistryContext, quay *v1.QuayRegistr
 		return nil, err
 	}
 
+	if ctx.DbRootPw == "" {
+		rootpw, err := generateRandomString(32)
```

Comment:
Nit: maybe `rootPw` to be consistent with the rest of the PR

Reply:
Yes, you are right. This was a copy and paste from a branch where I am working in refactoring this. I guess we can live with this until we land that other PR of mine.